src/hooks/useMedia.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
```

Comment:
I understand this is called in usePrefersDarkMode hook.
If you can explain the functionality of this hook (useMedia) then it would be great for future developers to quickly understand.